subsys/nrf_security/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ARM_MBEDTLS_PATH ${ZEPHYR_MBEDTLS_MODULE_DIR})
 # builds or when MBEDTLS_USE_PSA_CRYPTO is enabled (e.g. for TLS/DTLS
 # and x.509 support) Note: This configuration is internal and may be
 # removed with a new mbed TLS version
-set(CONFIG_MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER     False)
+set(CONFIG_MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER     True)
```

Comment:
Comment above is not reflecting the change in this line.

However I don't like modifying Kconfig in cmake, so I suggest instead to remove this completely and set the Kconfig symbol to y instead.
https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/nrf_security/Kconfig.psa#L17

Reply:
Okay sounds fair, then I remove this block completely and just set the default to y